protos/backend_service.proto

```diff
@@ -127,7 +130,7 @@ message AbandonActivityWorkItemResponse {
 message CompleteOrchestrationWorkItemRequest {
     // The completion token that was provided when the work item was fetched.
     string completionToken = 1;
-    google.protobuf.StringValue newExecutionId = 2;
+    OrchestrationInstance instance = 2;
```

Comment:
This is correcting a mistake that was made in the initial commit for this message contract. Note that DTS-related artifacts (which were not built from this copy, but from another copy) already have this change.